google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/descriptor/AppEngineWebSchemaProvider.java

```diff
@@ -73,10 +74,14 @@ public XmlFile getSchema(@NotNull @NonNls String url, @Nullable Module module,
       if (facet != null) {
         final File file;
         if (isApplicationXmlFile(baseFile)) {
-          file = facet.getSdk().getApplicationSchemeFile();
+          file = CloudSdkService.getInstance().getApplicationSchemeFile();
         } else {
-          file = facet.getSdk().getWebSchemeFile();
+          file = CloudSdkService.getInstance().getWebSchemeFile();
```

Comment:
Yea, we currently do not support ear deployments. The individual modules need to be deployed separately so I agree this can be removed if that's what it was for.